vim_snippet.vim

```diff
@@ -0,0 +1,25 @@
+function! s:RunBlackMacchiato(visual) range
```

Comment:
can this also be run from the `:` command prompt?

if so, maybe `BlackMacchiato` is a more discoverable name?

if not, a `:BlackMacchiato` command (it has tab completion i hope) that behaves the same as this function would be useful, and simpler to use in case users don't want to make a dedicated key binding.

Reply:
No, the function cannot be called as a command. For this to also work as a command, I will need to rewrite the function.

Reply:
I need to learn how to do it 😁

Reply:
Done :)